README.md

```diff
@@ -56,22 +89,57 @@ This variable is set to 0 by default.
 
 ### Signals
 
-Signals are the same as normal nginx.
+Signals are the same as normal NGINX.
```

Comment:
I believe NGINX is not used everywhere: https://www.nginx.com/resources/wiki/
Also, https://github.com/nginxinc/nginx-wiki/issues/146

Apparently some time ago [it was different](https://forum.nginx.org/read.php?2,255874,255879#msg-255879), but NGINX seems the right spelling for both the company and the product now. Unless there's difference between open source and supported, but I don't think it's the case either. We use NGINX in our [support site](https://support.3scale.net).